### PR TITLE
Further updates to tweak UX:

### PR DESCRIPTION
**Pull request by bbbco**
Here are a slew of updates I made to the extension to address some UX issues I've experienced after  using it more extensively this past month or two:

- Don't show unnecessary logs in console
- Tweak UX handling of filter to provide better feedback upon JQ filter execution and errors by appropriately changing the border of the filter textbox
- Better UI under Load Filters modal
    - Add confirmation dialog when deleting filters
    - Make [x] into an actual button that only displays when hovering over the filter
    - Other CSS tweaks

### CODE REVIEW

**Comment by patandrick** [comment]
Thanks for your contribution with this great PR! 
I'll test in the next few days. 
Could you please sign-off the commit for the dco check meanwhile?

**Comment by bbbco** [comment]
Yep, sorry, I forgot about the DCO. Signed.

File: src/ts/App/inputOutput.tsx [hunk 1]
```diff
@@ -30,6 +30,10 @@ export default class InputOutput extends Component<InputOutputProps, InputOutput
     const { hideUnfilteredJson } = this.state;
     const prettyInput = prettyJson(inputJson);
     const prettyOutput = prettyJson(outputJson);
+    // Remove extension specific options before passing to Brace
```
Comment (mrandi) [inline src/ts/App/inputOutput.tsx]: 👍

**Comment by mrandi** [comment]
@bbbco that's great! thx!

**Comment by mrandi** [comment]
👍

**Comment by harti2006** [comment]
:+1:

**Comment by mrandi** [comment]
Hi @bbbco,

I made some more detailed manual test after merging everything in master.
Seems that we have few issue with the changes (don't think they are all related to this pr)
https://github.com/zalando-incubator/bro-q/issues/350
https://github.com/zalando-incubator/bro-q/issues/351
https://github.com/zalando-incubator/bro-q/issues/352
Will try to go through this as soon as I can, just wanted to let you know.

Thanks again!

**Pull request merged**